operations/operations/publishing/publishing-guidelines/documentation-style-guide.md

```diff
@@ -0,0 +1,272 @@
+document up into logical sections. Each section should have a title, and the title should relate 
+to the content of the section. Use sentence case for the section title.
+
+A section title is not required if you have only one section.
+
+### Document title
+
+Use a title that accurately reflects the content of the document. People scan the table of contents 
+looking for answers; it's often faster than using the built-in search engine.
+
+Use title case for document titles (e.g. "This is an article about documentation").
+
+### Abstract
+
+The abstract provides a high-level overview of the content, includes context, and outlines the value or goal to the reader. 
+It often helps to write the abstract first, as it provides a framework for the content that follows. Keep in mind that the 
+description is displayed in the search results along with the page title. People read the description to 
+help them decide if the document is the one that they want.
+
+### Table of contents
+
+If the document contains more than four sections or subsections, add a table of contents to help the user
+navigate the document. The table of contents should be added below the abstract. 
+
+### Introductory paragraph
+
+Each page should have an introduction that acts as a short description of the document. The short description
+should be a single paragraph of no more than three sentences. 
+
+## Using ReStructuredText markup (rst)
+
+The reStructuredText specification allows for a certain degree of flexibility in 
+markup to achieve your goals. For example, you can use any one of more than a dozen characters for 
+section title underlines, and you have the option of using an overline in addition to an underline.
+
+The majority of Mattermost technical documentation is written in `.rst`. However, there are some instances
+where Markdown is used. 
+
+Use the following conventions when writing in `.rst`. 
+
+### Page titles
+
+Underline page titles using `=`, with no overline. Underlines should be as long as the title text. For example:
+
+```
+  Document title
+  ==============
+```
+
+### Section titles
+
+Underline using `-` for section titles. For example:
+
+```
+  Section title
+  -------------
+``` 
+
+Underline subsections using `~` for the first subsection level, and `^` for the second subsection level. For example:
+
+```
+  Subsection one
+  ~~~~~~~~~~~~~~
+
+  Subsection two
+  ^^^^^^^^^^^^^^
+``` 
+
+### Table of contents
+
+Insert a table of contents into a document using the following format:
+
+```
+  .. contents::
+      :backlinks: top
+```  
+    
+### Text formatting
+-----------------
+
+Use highlighting of text to visually set off words and phrases that are important to readers. Content that should be highlighted includes file names, UI controls, and window titles. The following table has a comprehensive list with examples.
+
+| Text          | Format        | Example             |
+| ------------- |:-------------:| -------------------:|
```

Comment:
```suggestion
| ------------- |:-------------:| --------------------|
```

Not sure if the "Example" heading is on the RHS because of this or if it's going to look ok when merged without any changes.